CMakeLists.txt

```diff
@@ -76,9 +76,9 @@ endif()
 set_if_not_defined(NIFTI_PACKAGE_PREFIX "")
 set(PACKAGE_NAME ${NIFTI_PACKAGE_PREFIX}NIFTI)
 set(NIFTI_INSTALL_EXPORT_NAME ${PACKAGE_NAME}Targets)
-set_if_not_defined(NIFTI_INSTALL_RUNTIME_DIR "${CMAKE_INSTALL_PREFIX}/bin")
-set_if_not_defined(NIFTI_INSTALL_LIBRARY_DIR "${CMAKE_INSTALL_PREFIX}/lib")
-set_if_not_defined(NIFTI_INSTALL_INCLUDE_DIR "${CMAKE_INSTALL_PREFIX}/include")
+set_if_not_defined(NIFTI_INSTALL_RUNTIME_DIR "${CMAKE_INSTALL_PREFIX}/bin/${PACKAGE_NAME}")
```

Comment:
This seems like a very bad idea.  Can you elaborate on the problem encountered?

Also "//' should be "/"

Reply:
I think I have fixed this now